RoundStartFreezeTimeManager.cs

```diff
@@ -84,6 +98,22 @@ private void CountdownTimerTick()
         {
             player.PrintToCenter(timeLeft > 0 ? $"Round begins in {timeLeft:0.0} seconds!" : "Round start!");
         }
+
+    }
+
+    private void PlaySound()
```

Comment:
Could do with calling it something like `PlaySoundTimer` just to make it more clear its used with a Timer

Reply:
yeah il do that